### ISSUE TITLE
[feature]: In walkthroughs we should also add a column of how to fork a repo 

### ISSUE THREAD

**Issue by kantakshay** [OpenSourcePal/OpenSourcePal#11]
In walkthroughs we should also add a column of how to fork a repo like, how to clone a repo is added

**Comment by Dun-sin** [comment]
do you want to take this?

**Comment by kantakshay** [comment]
> do you want to take this?

yes, I will take it. can you help me with this step Fill in the .env with the .env_sample, What needs to be filled?

**Comment by Dun-sin** [comment]
> > do you want to take this?
> 
> yes, I will take it. can you help me with this step Fill in the .env with the .env_sample, What needs to be filled?

Can you ask this again on discord? So I can give you a more comprehensive answer.
